/Users/zhousicheng/opt/miniconda3/envs/parsl_env310/bin/process_worker_pool.py:4: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  __import__('pkg_resources').require('parsl==1.3.0.dev0')
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
DEBUG:parsl.monitoring.resource_monitor:Sending intermediate resource message
ERROR:root:Exception reading IO counters for main process. Recorded IO usage may be incomplete
Traceback (most recent call last):
  File "/Users/zhousicheng/Desktop/resilient/parsl_resilient/parsl/monitoring/resource_monitor.py", line 67, in measure_resource_utilization
    d['psutil_process_disk_write'] = proc.io_counters().write_chars
AttributeError: 'Process' object has no attribute 'io_counters'
/Users/zhousicheng/opt/miniconda3/envs/parsl_env310/lib/python3.10/multiprocessing/resource_tracker.py:224: UserWarning: resource_tracker: There appear to be 7 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
